src/test/java/com/nerodesk/takes/doc/TkReadTest.java

```diff
@@ -59,16 +60,20 @@ public void readsFileContent() throws Exception {
         final Doc doc = base.user("urn:test:1").docs().doc("hey");
         final String input = "hello, world!";
         doc.write(IOUtils.toInputStream(input), input.getBytes().length);
+        final Response resp = new TkRead(base).act(
+            new RqWithTester(
+                new RqFake("GET", "/?file=hey")
+            )
+        );
         MatcherAssert.assertThat(
-            new RsPrint(
-                new TkRead(base).act(
-                    new RqWithTester(
-                        new RqFake("GET", "/?file=hey")
-                    )
-                )
-            ).printBody(),
+            new RsPrint(resp).printBody(),
             Matchers.endsWith("world!")
         );
+        MatcherAssert.assertThat(
+            new RsPrint(resp).printHead(),
+            //@checkstyle LineLengthCheck (1 lines)
+            Matchers.containsString("Content-Disposition: attachment; filename=\"hey\"")
```

Comment:
@dmzaytsev would we need checkstyle suppression if put the string to the new line?
```
Matchers.containsString(
    "Content-Disposition: attachment; filename=\"hey\""
)
```
?